constrained_ik/CHANGELOG.rst

```diff
@@ -0,0 +1,11 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package constrained_ik
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.1 (2017-03-24)
+------------------
+
```

Comment:
I think if there were no changes we either add *No Changes* or something along those lines.